input/templates/partials/catalog/mix-match-product3.json

```diff
@@ -0,0 +1,259 @@
+{
+    "list" : [
```

Comment:
How exactly? We can simply add more attributes to `product-data` if it's just a few attributes. That's what will be done between line items and products.